.github/workflows/deployment-development.yml

```diff
@@ -0,0 +1,54 @@
+# Deployment workflow to be used for Jira reporting
+name: Deploy alpha to Development
+
+on:
+  push:
+    branches: ["alpha"]
+
+  workflow_dispatch:
```

Comment:
Please delete this, as we don't want to run it manually

Reply:
I noticed that some times, the workflow may fail due to github connection error or else. That is why I added this one

Reply:
Ah, so now we would be able to manually run it as well?

Reply:
yes, As I'm using github.ref_name to get the tag name and this variable is accessible for all trigger events